content/blog/survey-02/index.md

```diff
@@ -0,0 +1,211 @@
+While votes for the latter did not reach a majority, the written feedback we got included quite a few requests for more "personality" in the newsletter.
+
+## Contributions
+
+![Contributions barplot](contributions.png)
+
+The majority of readers (61.5%) have not yet contributed to the newsletter and 26.9% contributed 2-5 times. 
+Only 3.8% contributed exactly once, while the rest (7.7%) are heavy contributors, helping us out more than five times.
+
+While it might seem weird that more people contributed 2-5 times than exactly once,
+keep in mind that the former is the sum of people who contributed twice, thrice, four times, and five times. 
+The reason we binned these together is that we are interested in the following categories:
+- Pure readers
+- People who contributed once and then stopped
+- People who contributed a few times
+- People who are regular contributors
+
+We are happy to see that people who contributed once seem to continue contributing in the future.
+
+## Ease
+
+![Ease barplot](ease.png)
+
+The mean ease of contributing is 3.0, and the median is 3. The 95% confidence interval is [2.5, 3.6] using a standard error of 0.26.
+
+Readers generally feel neutral about the ease of contributing to the newsletter (the p-value of a t-test of mean 3 is 0.87).
+
+We can do better here, but we are not sure yet how. 
+We'd love to hear your ideas on [GitHub](https://github.com/rust-gamedev/rust-gamedev.github.io/issues/1519) or on Discord (ping @janhohenheim).
+
+## Keeping up with the newsletter
+
+![Keeping up barplot](informed.png)
+
+This was a multiple-choice question. The most popular source of information about the newsletter is RSS (27.5%). 
+If we add the choices for the official Rust GameDev Discord server (21.7%) and other Discord servers (11.6%), 
+Discord in general becomes the leading source of information (33.3%), taking up nearly exactly a third of all votes.
+
+We can see the shift from X / Twitter to Mastodon reported by many OSS communities in our readers as well. 
+Lemmy is not looking popular as an alternative to Reddit yet, with no reader reporting it as a source of information.
+
+The "Email" option in the survey is meant for people who have set up some kind of email alerts manually.
+
+Per written feedback, a lot of people want to see proper email subscriptions implemented. While this was a goal for this month, 
+we have not managed to implement it yet because of personal things that came up for Jan Hohenheim, who volunteered to implement it.
+We will try to get this done for the next newsletter.
+
+## What is going well
+
+![What is going well wordcloud](like.png)
+
+This was a free-text question. The above is a word cloud of the answers with some obvious words like "game" or "newsletter" removed.
+Note that the inclusion of the word "AI" is misleading, as it was only mentioned in texts like "I like that we don't use AI, please do".
```

Comment:
The soft-quoted sentence finishes weirdly.

also: is it an exact quote or an aggregation? (the use of « like » might infer approximation in the quote ?)

Reply:
A whoops., thanks.
It's not an aggregation of the following exact quotes posted to the "What do you like?" question:
- "Stop using AI."
- "No AI please"
- "Please don't start generating articles with LLMs"

Reply:
Better? [`d4972ba` (#1520)](https://github.com/rust-gamedev/rust-gamedev.github.io/pull/1520/commits/d4972baac6b3e5357578d3ffe7170dd2dfffe91c)